Sources/MenuTransitioningDelegate.swift

```diff
@@ -21,7 +21,7 @@ import Foundation
 /**
  Delegate of menu transitioning actions.
  */
-final class MenuTransitioningDelegate: NSObject {
+public final class MenuTransitioningDelegate: NSObject {
```

Comment:
Hey @AndreyArzhannikov 
Yeah, the thing is without `public` there you get `Property cannot be declared public because its type uses an internal type` on `Menu` protocol